### ISSUE TITLE
Improve consistency of "Use custom description"

### ISSUE THREAD

**Issue by jindrichmynarz** [UnifiedViews/Plugins#341]
"Use custom description" in some DPUs is turned on by default (e.g., e-sparqlEndpoint) and turned off by default in others (e.g., t-sparqlUpdate). This should be made more consistent at least across the core DPUs.

**Comment by tomas-knap** [comment]
Agreed. I suggest to have it unchecked everywhere (so that default generated description is used by default). Also at the same time, it should be tested that every Core DPU generates reasonable default descriptions by calling getDescription() method (see e.g. https://github.com/UnifiedViews/Plugins/blob/master/t-rdfToFiles/src/main/java/eu/unifiedviews/plugins/transformer/rdftofiles/RdfToFilesVaadinDialog.java#L140 )
